src/figuresClasses.ts

```diff
@@ -1,11 +1,97 @@
-export interface Figure {}
+type Shape = 'triangle' | 'rectangle' | 'circle';
+type Color = `red` | `green` | `blue`;
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
+}
+
+type ValidateTriangle = (a: number, b: number, c: number) => boolean;
+
+const validateTriangle: ValidateTriangle = (a, b, c) => {
+  const arrFromNum: number[] = [a, b, c];
+  const maxLength: number = Math.max(...arrFromNum);
+  const sum = arrFromNum
+    .filter((num: number) => num !== maxLength)
+    .reduce((acc: number, elem: number) => acc + elem, 0);
+
+  return maxLength >= sum;
+};
```

Comment:
Thank you!) I was scared to use it inside because 'classes' is my weakness and I thought that method would not be seen by the program inside the constructor. For instance, it would be undefined of reference error, something like this)

Reply:
This one doesn't work( 
![image](https://github.com/mate-academy/ts_figures_classes/assets/102412814/52ec0c7c-1770-4f3e-9899-e35ea11110d1)